buildAndReleaseTask/installers/pulumi.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as tl from "azure-pipelines-task-lib/task";
+import * as lib from "azure-pipelines-tool-lib";
+import * as path from "path";
+
```

Comment:
A doc comment here would be good to explain what this function is meant to do.

Reply:
done